static/js/stream_popover.js

```diff
@@ -262,6 +263,21 @@ function build_starred_messages_popover(e) {
 
 }
 
+function build_move_topic_to_stream_popover(e, current_stream_id, topic_name) {
+    // TODO: Add support for keyboard-alphabet navigation. Some orgs have large no. of
```

Comment:
"large number" -- we generally avoid abbreviations in the codebase and comments, since they can be harder to read.